.github/workflows/_build.yml

```diff
@@ -118,11 +118,28 @@ jobs:
           disable-animations: true
           script: ./gradlew :Backpack:connected${{ env.config }}AndroidTest :backpack-compose:connected${{ env.config }}AndroidTest :backpack-common:connected${{ env.config }}AndroidTest
 
+  CheckCommitMessage:
```

Comment:
CI checks out a version that merges main into the branch, so need to specifically check out the correct version to get the latest commit message